Microsoft.Toolkit.Uwp.UI.Controls.Layout/ListDetailsView/ListDetailsView.cs

```diff
@@ -62,258 +60,123 @@ public ListDetailsView()
         }
 
         /// <summary>
-        /// Invoked whenever application code or internal processes (such as a rebuilding layout pass) call
-        /// ApplyTemplate. In simplest terms, this means the method is called just before a UI element displays
-        /// in your app. Override this method to influence the default post-template logic of a class.
+        /// Updates the visual state of the control.
         /// </summary>
-        protected override void OnApplyTemplate()
+        /// <param name="animate">False to skip animations.</param>
+        private void SetVisualState(bool animate)
```

Comment:
Methods have been moved around making review very hard. private methods should be below public/protected methods

Reply:
Fixed the method order.